Source/Engine/Physics/Physics.cpp

```diff
@@ -422,45 +324,8 @@ bool PhysicsService::Init()
     }
 #endif
 
-    // Create scene description
-    PxSceneDesc sceneDesc(CPhysX->getTolerancesScale());
-    sceneDesc.gravity = C2P(settings.DefaultGravity);
-    sceneDesc.flags |= PxSceneFlag::eENABLE_ACTIVE_ACTORS;
-    if (!settings.DisableCCD)
-        sceneDesc.flags |= PxSceneFlag::eENABLE_CCD;
-    if (settings.EnableAdaptiveForce)
-        sceneDesc.flags |= PxSceneFlag::eADAPTIVE_FORCE;
-    sceneDesc.simulationEventCallback = &EventsCallback;
-    sceneDesc.filterShader = FilterShader;
-    sceneDesc.bounceThresholdVelocity = settings.BounceThresholdVelocity;
-    if (sceneDesc.cpuDispatcher == nullptr)
-    {
-        CpuDispatcher = PxDefaultCpuDispatcherCreate(Math::Clamp<uint32>(cpuInfo.ProcessorCoreCount - 1, 1, 4));
-        CHECK_INIT(CpuDispatcher, "PxDefaultCpuDispatcherCreate failed!");
-        sceneDesc.cpuDispatcher = CpuDispatcher;
-    }
-    if (sceneDesc.filterShader == nullptr)
-    {
-        sceneDesc.filterShader = PhysXDefaultFilterShader;
-    }
-
-    // Create scene
-    PhysicsScene = CPhysX->createScene(sceneDesc);
-    CHECK_INIT(PhysicsScene, "createScene failed!");
-#if WITH_PVD
-    auto pvdClient = PhysicsScene->getScenePvdClient();
-    if (pvdClient)
-    {
-        pvdClient->setScenePvdFlags(PxPvdSceneFlag::eTRANSMIT_CONSTRAINTS | PxPvdSceneFlag::eTRANSMIT_SCENEQUERIES | PxPvdSceneFlag::eTRANSMIT_CONTACTS);
-    }
-    else
-    {
-        LOG(Info, "Missing PVD client scene.");
-    }
-#endif
-
-    // Init characters controller
-    ControllerManager = PxCreateControllerManager(*PhysicsScene);
+    Physics::DefaultScene = new PhysicsScene(String("Default"), settings, cpuInfo);
```

Comment:
Use `New<PhysicsScene>(TEXT("Default"))` as a default memory allocator in Flax.

Reply:
Thanks, I replaced this with a call to `Physics::FindOrCreateScene` to centralize where `PhysicsScene`'s are created from and replaced the `new ` with the flax allocator.